lib/Events/MessagesDeletedEvent.php

```diff
@@ -57,7 +57,7 @@ public function getMailbox(): Mailbox {
 		return $this->mailbox;
 	}
 
-	public function getMessageId(): int {
-		return $this->messageId;
+	public function getMessageIds(): array {
```

Comment:
```suggestion
	public function getMessageUids(): array {
```

because I was just confused between id vs uid

Reply:
done